packages/flutter_tools/lib/src/android/gradle_errors.dart

```diff
@@ -729,3 +730,26 @@ https://github.com/flutter/flutter/issues/156304''', title: _boxTitle);
   },
   eventLabel: 'java21-and-source-compatibility',
 );
+
+final RegExp _missingNdkSourcePropertiesRegexp = RegExp(r'NDK at ((?:/|[a-zA-Z]:\\).+?) did not have a source\.properties file', multiLine: true);
```

Comment:
The forward slash might need to be escaped, consider updating the unit tests to cover this case:

```suggestion
final RegExp _missingNdkSourcePropertiesRegexp = RegExp(r'NDK at ((?:\/|[a-zA-Z]:\\).+?) did not have a source\.properties file', multiLine: true);
```